src/select_data_without_data_attributes.coffee

```diff
@@ -0,0 +1,21 @@
+define [
+  'jquery'
+  './text_formatter'
+], (
+  $
+  textFormatter
+) ->
+  subgroup: ($elem) ->
+    $elem.closest('[id]').attr('id') or ''
+
+  item: ($elem) ->
+    value = $elem.find(':selected').val() or ''
+    textFormatter.replaceDoubleByteChars(value)
+
+  value: ($elem) ->
```

Comment:
Can this and the code starting at line 11 be refactored? It looks like they both do the same thing.